src/aave-v3/SupplyVault.sol

```diff
@@ -168,18 +160,10 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
         );
 
         UserRewardsData memory rewards = userRewards[_rewardToken][_user];
```

Comment:
this variable could be read from storage inside `_getUnaccruedReward` (because it's read everywhere we are using `_getUnaccruedReward`) ; but it requires renaming `_getUnaccruedReward` to `_getUnclaimedReward`

Also, isn't it missing the plural? (ie `_getUnclaimedRewards`)

Reply:
_getUnaccruedReward only gets the reward for one specified token. The intention is for this to be called in a loop for each reward token.

Reply:
My main point was: given that `_getUnaccruedReward` takes `_user` and `_rewardToken` as parameters, it should be interesting to access the storage value `userRewards[_rewardToken][_user]` inside it

My secondary point was: `reward` not plural is valid english?

Reply:
Oh, I see. So you are proposing that userRewards.unclaimed be integrated into getUnaccruedReward and change the naming to getUnclaimedReward.

I didn't want to use plural because I wanted to make clear in the naming that this was for a single reward token. Maybe it's not perfect English, but imo it provides clarity so I think singular is appropriate.